### ISSUE TITLE
Problem with results of podman inspect --format

### ISSUE THREAD

**Issue by SergeyBear** [containers/libpod#2462]
/kind bug

**Description**

Podman inspect does not return exit status 1 nor error if element in --format is absent.
Looks like https://github.com/containers/libpod/issues/2159 is back again.

**Steps to reproduce the issue:**

podman pull busybox
podman inspect --format '{{.ContainerConfig.Labels.123}}' busybox || echo error

**Describe the results you received:**

exit code 0 and empty output

**Describe the results you expected:**

exit code 1 and some notification

**Output of `podman version`:**

```
Version:       0.12.1.2
Go Version:    go1.10.3
OS/Arch:       linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.10.3
  podman version: 0.12.1.2
host:
  BuildahVersion: 1.6-dev
  Conmon:
    package: podman-0.12.1.2-2.git9551f6b.el7.centos.x86_64
    path: /usr/libexec/podman/conmon
    version: 'conmon version 1.12.0-dev, commit: b909c9e1a3e8f14d5694a118fb9c0c0325a31d4f-dirty'
  Distribution:
    distribution: '"centos"'
    version: "7"
  MemFree: 1239961600
  MemTotal: 1927245824
  OCIRuntime:
    package: runc-1.0.0-59.dev.git2abd837.el7.centos.x86_64
    path: /usr/bin/runc
    version: 'runc version spec: 1.0.0'
  SwapFree: 0
  SwapTotal: 0
  arch: amd64
  cpus: 2
  hostname: centos7
  kernel: 3.10.0-957.5.1.el7.x86_64
  os: linux
  rootless: false
  uptime: 1h 41m 22.56s (Approximately 0.04 days)
insecure registries:
  registries: []
registries:
  registries:
  - registry.access.redhat.com
  - docker.io
  - registry.fedoraproject.org
  - quay.io
  - registry.centos.org
store:
  ContainerStore:
    number: 0
  GraphDriverName: overlay
  GraphOptions: null
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 3
  RunRoot: /var/run/containers/storage

```

**Comment by haircommander** [comment]
This has since been fixed! If you upgrade podman to at least v1.0 you shouldn't get this problem

**Comment by SergeyBear** [comment]
Unfortunately in Centos 7 only podman 0.12 is available. Using fedora or building from source is not a production way.
Any plans to backport fixes from 1.0 to 0.12 or porting podman 1.0 to Centos 7?
